frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx

```diff
@@ -189,6 +189,7 @@ export function PropertyValue({
             loading={options[propertyKey]?.status === 'loading'}
             propertyKey={propertyKey}
             {...commonInputProps}
+            className="property-filters-property-value"
```

Comment:
Obviously, ideally, we replace the property value component with lemon ones too, BUT

1) Property values code is not easy to sort through and refactor
2) This is blocking https://github.com/PostHog/posthog/pull/11490 and I don't want to rabbit hole with this
3) Cohorts does the same thing currently with the applied class
4) We don't have consistent styling in general for our property filters/values and I don't think this PR is the place to fix it all in one go

Reply:
This is regarding the last box in the filters. For now I'm imitating the desired look with CSS (using the same styling from cohorts property values)

Reply:
> Property values code is not easy to sort through and refactor

That's why we employ developers: to do the things that are not easy ;). We are talking about replacing one multiselect, so... 🤔 

Overall, I'm really not a fan of this approach, as we're explicitly adding code we know we need to rewrite soon. However the refactor does make many things cleaner, so I'm willing to let it slide (and then change it myself probably, as I was thinking of working in this area).

However, shouldn't these custom classes be in `PropertyValue.scss`, not `global.scss`? That'll at least keep the custom CSS close enough that it can be removed with the next refactor. Hiding it in `global` means it'll be around far longer than the component.